### PR TITLE
Add ascii art to the main repo

### PR DESCRIPTION
**Pull request by Sumith1896**
:smile:

### CODE REVIEW

File: symengine/printer.cpp [hunk 1]
```diff
@@ -5,6 +5,16 @@
 
 namespace SymEngine {
 
+void ascii_art() {
+    std::cout << "                                         \n"
```
Comment (abinashmeher999) [inline symengine/printer.cpp]: This line can be removed, right?

**Comment by abinashmeher999** [comment]
Looks cool! :smile:

**Comment by CodeMaxx** [comment]
Nice...We should have a Symengine logo too!

**Comment by certik** [comment]
Change it to print it to a string, not stdout, so that the wrappers can
print it in Python as well.

**Comment by Sumith1896** [comment]
Done. Thanks @certik @abinashmeher999

**Comment by certik** [comment]
+1

**Comment by abinashmeher999** [comment]
+1

File: symengine/printer.cpp [hunk 2]
```diff
@@ -5,6 +5,16 @@
 
 namespace SymEngine {
 
+std::string ascii_art() {
+
```
Comment (abinashmeher999) [inline symengine/printer.cpp]: @Sumith1896 Feels like there is a trailing space here.

Comment (Sumith1896) [inline symengine/printer.cpp]: I better remove the whole line!

**Comment by Sumith1896** [comment]
Ready to merge.

**Comment by certik** [comment]
Thanks!

**Comment by Sumith1896** [comment]
Thanks @certik

**Comment by certik** [comment]
So in this comment https://github.com/symengine/symengine/pull/414#discussion_r32447845 I proposed several different versions of the art, some of which I actually like better. Make sure you also follow the links ([1](https://gist.github.com/certik/a78cef8105f0c4b34e41) and [2](http://patorjk.com/software/taag/#p=testall&f=Graffiti&t=SymEngine)).

I think I like the `big` and `doom` better (I prefer the `doom`, but taking the `S` from `big`).

**Comment by certik** [comment]
We should also probably have an ascii version and a unicode version. It's hard to find unicode generators, but I found e.g. http://fsymbols.com/generators/carty/
```
╭━━╮╱╱╱╱╭━╮╱╱╱╱╭╮
┃━━╋┳┳━━┫┳╋━┳┳━╋╋━┳┳━╮
┣━━┃┃┃┃┃┃┻┫┃┃┃╋┃┃┃┃┃┻┫
╰━━╋╮┣┻┻┻━┻┻━╋╮┣┻┻━┻━╯
╱╱╱╰━╯╱╱╱╱╱╱╱╰━╯

╭━━━╮╱╱╱╱╱╱╭━━━╮
┃╭━╮┃╱╱╱╱╱╱┃╭━━╯
┃╰━━┳╮╱╭┳╮╭┫╰━━┳━╮╭━━┳┳━╮╭━━╮
╰━━╮┃┃╱┃┃╰╯┃╭━━┫╭╮┫╭╮┣┫╭╮┫┃━┫
┃╰━╯┃╰━╯┃┃┃┃╰━━┫┃┃┃╰╯┃┃┃┃┃┃━┫
╰━━━┻━╮╭┻┻┻┻━━━┻╯╰┻━╮┣┻╯╰┻━━╯
╱╱╱╱╭━╯┃╱╱╱╱╱╱╱╱╱╱╭━╯┃
╱╱╱╱╰━━╯╱╱╱╱╱╱╱╱╱╱╰━━╯


╔═══╗──────╔═══╗
║╔═╗║──────║╔══╝
║╚══╦╗─╔╦╗╔╣╚══╦═╗╔══╦╦═╗╔══╗
╚══╗║║─║║╚╝║╔══╣╔╗╣╔╗╠╣╔╗╣║═╣
║╚═╝║╚═╝║║║║╚══╣║║║╚╝║║║║║║═╣
╚═══╩═╗╔╩╩╩╩═══╩╝╚╩═╗╠╩╝╚╩══╝
────╔═╝║──────────╔═╝║
────╚══╝──────────╚══╝


┏━━━┓╋╋╋╋╋╋┏━━━┓
┃┏━┓┃╋╋╋╋╋╋┃┏━━┛
┃┗━━┳┓╋┏┳┓┏┫┗━━┳━┓┏━━┳┳━┓┏━━┓
┗━━┓┃┃╋┃┃┗┛┃┏━━┫┏┓┫┏┓┣┫┏┓┫┃━┫
┃┗━┛┃┗━┛┃┃┃┃┗━━┫┃┃┃┗┛┃┃┃┃┃┃━┫
┗━━━┻━┓┏┻┻┻┻━━━┻┛┗┻━┓┣┻┛┗┻━━┛
╋╋╋╋┏━┛┃╋╋╋╋╋╋╋╋╋╋┏━┛┃
╋╋╋╋┗━━┛╋╋╋╋╋╋╋╋╋╋┗━━┛
```

**Comment by abinashmeher999** [comment]
@certik The `S` in `doom` might not look very clean on the browser, but it looks great on the console.
Have a look! This is how it looks like in vim.
screenshot from 2016-02-19 02 19 47
For the unicode version have a look at [this](http://www.richard-h-clark.com/projects/block-art.html). We can have *SymEngine* written in a nice font then converted to png and then use this script to generate the unicode.

**Pull request merged**